hadoop-common-project/hadoop-common/src/site/markdown/filesystem/abortable.md

```diff
@@ -87,15 +87,14 @@ for example. output streams returned by the S3A FileSystem.
 
 The stream MUST implement `Abortable` and `StreamCapabilities`.
 
-```python
- if unsupported:
+```
```

Comment:
can you restore this. yes, it bears more relation to the Z specification language of Ince93 etc, but if you tag as Python then IDEs cover it. And Github doesn't really handle any of the formal languages. Nor do most developers, which is why this uses a python-esque syntax.

Reply:
Ah, I considered the python tags detrimental because my IDE (Intellij) had totally incorrect syntax highlights for these snippets, because they are not valid Python. Specifically, the `FS'` causes the IDE to think a string is being defined.
But I'm fine with re-applying the python tags.

Reply:
I know it's not well formed python, but neither are our java or xml snippets in the docs. consider it best-effort